pip_review/__main__.py

```diff
@@ -146,34 +148,25 @@ def latest_version(pkg_name, prerelease=False, silent=False):
     return parse_version(version), version
 
 
-def get_latest_versions(pkg_names, prerelease=False):
-    get_latest = partial(latest_version, prerelease=prerelease, silent=True)
+def get_latest_versions(pkg_names, pre_release=False):
+    get_latest = partial(latest_version, pre_release=pre_release, silent=True)
     versions = map(get_latest, pkg_names)
     return zip(pkg_names, versions)
 
 
-def get_installed_pkgs(local=False):
-    logger = logging.getLogger(u'pip-review')
-    command = 'pip freeze'
-    if packaging_version.parse(pip.__version__) >= packaging_version.parse('8.0.3'):
-        command += ' --all'
+def get_installed_pkgs(local=False, user=False):
+    command = ['pip', 'list', '--format=json']
     if local:
-        command += ' --local'
+        command.append('--local')
+    if user:
+        command.append('--user')
 
-    output = check_output(command).decode('utf-8')
-
-    for line in output.splitlines():
-        if not line or line.startswith('##'):
-            continue
-
-        if line.startswith('-e'):
-            name = line.split('#egg=', 1)[1]
-            if name.endswith('-dev'):
-                name = name[:-4]
-            yield name, 'dev', 'dev', True
-        else:
-            name, version = line.split('==')
-            yield name, parse_version(version), version, False
```

Comment:
As I said in my comment to line 159->158, you will need parsing lines like the above to interpret the output of `pip list`.